src/main/java/pl/byczazagroda/trackexpensesappbackend/config/WebSecurityConfig.java

```diff
@@ -14,9 +15,12 @@
 @EnableWebSecurity
 public class WebSecurityConfig {
 
+    private final ObjectMapper objectMapper;
+
     private final String secret;
 
-    public WebSecurityConfig(@Value("${jwt.secret}") String secret) {
+    public WebSecurityConfig(@Value("${jwt.secret}") String secret, ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
         this.secret = secret;
     }
 
```

Comment:
Please delete constructor. Annotation @Requiredargsconstructor resolve this. Moreover we don't have to modify constructor everytime when additional parameter is added.